test/remote/gateways/remote_borgun_test.rb

```diff
@@ -8,7 +8,7 @@ def setup
     @gateway = BorgunGateway.new(fixtures(:borgun))
 
     @amount = 100
-    @credit_card = credit_card('5587402000012011', year: 2018, month: 9, verification_value: 415)
+    @credit_card = credit_card('5587402000012011', year: 2022, month: 9, verification_value: 415)
```

Comment:
Would `Time.now.year + 2` work? That we don't have to update the year every couple of years?

Reply:
Tried that as well as `+ 1`, and kept getting an invalid card for some reason. Decided it wasn't worth the time to figure out why... _bought us a few years_

Reply:
:/ That's unfortunate. Thanks for trying 😄